src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3937,7 +3937,8 @@ internal void UpdateTypes(ExecutionContext context, bool updateOnly)
 
             if (errors.Count > 0)
             {
-                var allErrors = new StringBuilder('\n');
+                // Note: don't use '\n' ('char' convert implicitly to 'ushort' and 'int' and initialize 'StringBuilder' size)
```

Comment:
To be clear, a reasonable estimate is probably better than the default, and constant, or easily computed value, e.g. count*80 our something like that is reasonable. But again, it really doesn't matter here.

As for a CoreFX issue - it's up to you. I feel like they might discuss it, but reject it in favor of fewer APIs because it should if you properly test, you'd notice your character is missing.